lib/experimental/Collections/Filters/index.tsx

```diff
@@ -142,6 +145,11 @@ export function Filters<Definition extends FiltersDefinition>({
           </PopoverContent>
         </Popover>
 
+        <div className="rounded-sm bg-f1-background-secondary px-2 py-1.5">
+          Here should be the presets
```

Comment:
Not sure I love having this hardcoded even if it's temporary 😆 . We should be able to have an acceptable experience without presets, right?